frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -102,7 +110,9 @@ export function SelectEmbedTypePane({
         <SharingPaneButton
           title={t`Static embedding`}
           illustration={<StaticEmbeddingIllustration />}
-          onClick={goToNextStep}
+          onClick={isStaticEmbeddingDisabled ? undefined : goToNextStep}
+          isDisabled={isStaticEmbeddingDisabled}
+          disabledLink="/admin/settings/embedding-in-other-applications/standalone"
```

Comment:
these 2 props control the disabled state and the link that the bottom link will bring users to. All 3 embedding options are controlled the same way.
![Screenshot 2024-11-28 at 9 16 44 PM](https://github.com/user-attachments/assets/21bd0bd3-1d3b-4ea3-b44a-16491fe968c2)